OptimizelySDK/OptimizelyTests/OptimizelyTests-APIs/OptimizelyClientTests_ObjcOthers.m

```diff
@@ -0,0 +1,272 @@
+/****************************************************************************
+ * Copyright 2019, Optimizely, Inc. and contributors                        *
+ *                                                                          *
+ * Licensed under the Apache License, Version 2.0 (the "License");          *
+ * you may not use this file except in compliance with the License.         *
+ * You may obtain a copy of the License at                                  *
+ *                                                                          *
+ *    http://www.apache.org/licenses/LICENSE-2.0                            *
+ *                                                                          *
+ * Unless required by applicable law or agreed to in writing, software      *
+ * distributed under the License is distributed on an "AS IS" BASIS,        *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. *
+ * See the License for the specific language governing permissions and      *
+ * limitations under the License.                                           *
+ ***************************************************************************/
+
+
+#import <XCTest/XCTest.h>
+@import Optimizely;
+
+
+static NSString * const kExperimentKey = @"exp_with_audience";
+static NSString * const kVariationKey = @"a";
+static NSString * const kVariationOtherKey = @"b";
+static NSString * const kEventKey = @"event1";
+static NSString * const kFeatureKey = @"feature_1";
+
+static NSString * const kUserId = @"11111";
+static NSString * const kSdkKey = @"12345";
+
+@interface OptimizelyClientTests_ObjcOthers : XCTestCase
+@property(nonatomic) NSDictionary * attributes;
+@end
+
+
+// MARK: - Custom EventDispatcher
+
+@interface MockOPTEventDispatcher: NSObject <OPTEventDispatcher>
+@property(atomic, assign) int eventCount;
+@end
+
+@implementation MockOPTEventDispatcher
+- (id)init {
+    self = [super init];
+    _eventCount = 0;
+    return self;
+}
+
+- (void)dispatchEventWithEvent:(EventForDispatch * _Nonnull)event completionHandler:(void (^ _Nullable)(NSData * _Nullable, NSError * _Nullable))completionHandler {
+    self.eventCount++;
+    return;
+}
+
+- (void)flushEvents {
+    return;
+}
+@end
+
+@implementation OptimizelyClientTests_ObjcOthers
+
+// MARK: - Others
+
+- (void)testCustomNotificationCenter_Activate {
```

Comment:
this seems incorrectly named CustomNotificationCenter_Activate?

Reply:
The name "testCustomNotificationCenter_Activate" looks ok to me. Not clear what you mean here.

Reply:
there is no custom notification center.  it should be testNotificationCenter_Activate